src/plugins/DocsTargetBranch/docs_target_branch.ts

```diff
@@ -88,13 +96,16 @@ const wrongTargetBranchDetected = async (
     `Adding comment to ${context.payload.pull_request.number}: ${body}`
   );
 
-  promises.push(
-    context.github.issues.createComment(
-      context.issue({
-        body,
-      })
-    )
-  );
+  if (!currentLabels.includes("needs-rebase")) {
+    // If the label "needs-rebase" allready exsist we can assume that this action has run, and we should ignore it.
```

Comment:
```suggestion
    // If the label "needs-rebase" already exsist we can assume that this action has run, and we should ignore it.
```

Reply:
Shouldn't we just cancel the whole operation if it has this tag, assuming that all other work has also already been done?

Reply:
🤦